package.json

```diff
@@ -11,21 +11,24 @@
   ],
   "scripts": {
     "analyze": "size-limit --why",
-    "build": "dts build",
-    "lint": "dts lint --config \"./.prettierrc.yml\"",
-    "prepare": "dts build",
+    "build": "tsc",
```

Comment:
Just a single tsc command will not produce any esm artifacts right (see `module` field above). See example here: https://github.com/hyperlane-xyz/hyperlane-widgets/blob/main/package.json#L77